src/components/AnonymousChat/PostNotificationChannelItem.tsx

```diff
@@ -38,7 +39,7 @@ const PostNotificationChannelItem: (props: MessageChannelItemProps) => React.Rea
       commenterName = 'You';
       postNotificationPicture = firstComment?.reaction?.user?.data?.profile_pic_url;
     } else if (anonymousCommenterName) {
-      commenterName = `Anonymous ${capitalizeFirstText(anonymousCommenterName)}`;
+      commenterName = getOfficialAnonUsername(firstComment?.reaction?.data);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `getOfficialAnonUsername` function here is a good change. It ensures that the official anonymous username is used consistently across the application.
```diff
- commenterName = `Anonymous ${capitalizeFirstText(anonymousCommenterName)}`;
+ commenterName = getOfficialAnonUsername(firstComment?.reaction?.data);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->